README.md

```diff
@@ -145,6 +145,7 @@ Where `username` is the user's GitHub or Gitlab username, and `contribution` is
 - tutorial: [✅](# 'Tutorials')
 - userTesting: [📓](# 'User Testing')
 - video: [📹](# 'Videos')
+- maintenance [👷](# 'Maintenance')
```

Comment:
this change has in pr https://github.com/all-contributors/all-contributors-cli/pull/142